src/include/miopen/problem.hpp

```diff
@@ -47,11 +48,15 @@ struct Handle;
 struct Solution;
 struct FindOptions;
 
-namespace conv {
+namespace activ {
```

Comment:
```suggestion
namespace conv {
```

Reply:
I have sorted them alphabetically.

Reply:
Oh, sorry, I have mixed how I have messed this up.

Reply:
[resolved]